modules/gsuite_enabled/variables.tf

```diff
@@ -171,8 +171,8 @@ variable "disable_dependent_services" {
   type        = string
 }
 
-variable "shared_vpc_enabled" {
-  description = "If shared VPC should be used"
+variable "enable_shared_vpc_service_project" {
```

Comment:
Please leave the name unchanged on outer modules for backwards-compatibility.

Reply:
again, you asked to rename this variable. should I do it only in the root module? it's going to be confusing.